packages/SwingSet/test/test-vpid-kernel.js

```diff
@@ -608,3 +591,240 @@ for (const caseNum of [4, 5, 6, 7]) {
     });
   }
 }
+
+test(`kernel vpid handling crossing resolutions`, async t => {
```

Comment:
I think we can simplify this to use just a single vat, and build promise data up in the kernel directly (rather than using a separate vat to construct it). Let's walk through it today.

Reply:
did we write up a simpler test to exercise this?